mlir/utils/performance/perfRunner.py

```diff
@@ -837,7 +837,7 @@ def getFusionTestInfo(filename, paths: Paths):
     p2.stdout.close()
     output, _ = tuningKey.communicate()
     result = output.decode('utf-8').strip().split('\t')
-    testEntry = {'filename' : filename, 'testVector' : result[1], 'futName' : futName}
+    testEntry = {'filename' : filename, 'testVector' : result[2], 'futName' : futName}
```

Comment:
[PR#1145](https://github.com/ROCmSoftwarePlatform/rocMLIR/pull/1145) added 'numCU' to the problem description string, as a result the tuning script should skip it and extract the test vector from the third item.